core-bundle/src/Image/PictureFactory.php

```diff
@@ -157,9 +157,12 @@ private function createConfig(array|int|string|null $size): array
                 if (null !== $imageSizes) {
                     $options->setSkipIfDimensionsMatch((bool) $imageSizes->skipIfDimensionsMatch);
 
-                    if (!$imageSizes->preserveMetadata) {
+                    if ('metaDelete' === $imageSizes->preserveMetadata) {
```

Comment:
```suggestion
                    if ('delete' === $imageSizes->preserveMetadata) {
```
You changed the keys now I think, so they need to be changed here too. (There are three more usages in this file)